.gitignore

```diff
@@ -6,3 +6,4 @@
 /build/
 # Generated poms
 /maven/poms/**/pom.xml
+/.idea/
```

Comment:
See the comment at the top of the file and the link on gwtproject.org - maybe the readme should point that out too?